src/OSPSuite.UI/Extensions/BarItemExtensions.cs

```diff
@@ -11,15 +12,16 @@ public static BarItem UpdateIcon(this BarItem barItem, ApplicationIcon icon)
          if (icon == null)
             return barItem;
 
-         barItem.Glyph = transparentImage(icon, IconSizes.Size16x16);
-         barItem.LargeGlyph = transparentImage(icon, IconSizes.Size32x32);
+         //TODO This does not seeem to be supported yet
```

Comment:
what is the implication?

Reply:
I think it will resize automatically accordingly m we in fact don't need this anymore